JSON-connect/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dsccConnector": {
-    "production": "AKfycbwURHkauguMPzj2gJKaFRYR4z3qqNDDVRngu8YIJHGelfIZBDjbZCQu_TqARz0uaxtb",
-    "latest": "AKfycbzWcrFPGGlkUczJiPXmf7C1q4-8mBez7CwuH7Vinn4"
+    "production": "AKfycbzDZJEtN414a8F_vWQ3z9MHFJKXpXzGz8rq7De5jfPMmJz-k49Dpd4gbW5olVa0BvI",
```

Comment:
Please revert this change, we want to keep the current deployments.

Reply:
done